scripts/run_cargo_tests.sh

```diff
@@ -24,5 +24,5 @@ time cargo rustc --bin vhost_user_net --no-default-features --features "pci"  --
 time cargo clippy --all-targets --no-default-features --features "mmio" -- -D warnings
 time cargo rustc --bin cloud-hypervisor --no-default-features --features "mmio"  -- -D warnings
 time cargo rustc --bin vhost_user_net --no-default-features --features "mmio"  -- -D warnings
-time sh -c 'find . \( -name "*.rs" ! -wholename "*/out/*.rs" \) | xargs rustfmt --check'
+time sh -c 'find . -not \( -path ./build -prune \) \( -name "*.rs" ! -wholename "*/out/*.rs" \) | xargs rustfmt --check'
```

Comment:
Now that I finally setup up the workspace correctly we can just do `cargo fmt`

Reply:
Indeed, I just tested that it works. Will fix.

Reply:
Done.